templates/api.html

```diff
@@ -10,8 +10,11 @@ <h2>Available fields</h2>
 		<div style="float: right; width: 60%; padding-left: 10px;">
 			<h2>Examples</h2>
 			{% for example in $resource.examples %}
-				<h3 id="{{ example.id }}">{{ example.summary}}</h3>
-				<pre><code>curl -X {{ example.method }} \
+				<h3 id="{{ example.id }}">{{ example.summary }}</h3>
+        {% if example.description %}
+          <p>{{ example.description }}</p>
+        {% endif %}
+        <pre><code>curl -X {{ example.method }} \
```

Comment:
The indentation got wrong - mixed spaces and tabs. Looks like the original file used tabs, and the PR added spaces.

Reply:
Thanks for the catch looks like I have to adjust my editor settings to convert space to tabs by default